mpyg321/mpyg321.py

```diff
@@ -194,6 +220,8 @@ def set_player(self, player, audiodevice):
         args = "--remote" if self.player_version == "mpg123" else "-R test"
         args += " --audiodevice " + audiodevice if audiodevice else ""
         self.player = pexpect.spawn(str(player) + " " + args)
+        if self.player_version == "mpg123":
+            self.player.delaybeforesend = None
```

Comment:
What is this doing? Why do we need it for mpg123?

Reply:
- Volume is supported only in mpg123
- delaybeforesend property cause pexpect to wait 50ms before sending the command. In my setup, I have a MIDI keyboard and I use a slider to control the mpg123 volume, I have a logic that convert a MIDI message to a volume % for mpg123.
- I don't want the 50ms delay because changing the volume is not fluid, to change the volume from 0% to 100% of volume it takes 100 * 50ms (5 seconds) wich is a uge amount of time ! I
- setting this property to None change the behaviour with a realtime slider feel

Reply:
I understand thank you. However, I think we do not need the check here for the version. Let's say, someone wants to use the jump function in a reactive way (hot cues or something like this), the delay would be problematic. So I think it's better to keep the delaybeforesend no matter what version is in use :)

Reply:
I agree!